static/oauth.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
```

Comment:
Is all this necessary for an OAuth callback page? Usually those are very simple, but I don't know the details of what's going on so if it's needed go for it.

Is any of this generated somewhere else, or cut and pasted from somewhere else? If so it would be good to include a comment in both places so future updates aren't missed.

Reply:
It is copy and pasted from `index.html` and I have added a comment in both places (to keep them in sync).

This is not the ideal way of doing things, but I think cleaning up would require some good time effort.. so good for now.

Reply:
Yep, thanks for the comments!